### DISCUSSION TITLE
Week 10 🎉

### DISCUSSION THREAD

**Discussion by ColinFay** [ColinFay/ur-first-5k#394] [General]
Hey y'all! 

Yeay, you made it to the last week of this program! 

You should be able to do your 5k this weekend 🎉

Here is this week menu: 

```
── Day 1 
→ Rest
Rest day.

── Day 2 
→ W10D2-Intervals
• Warm up, 15 minutes.
• Run, steady pace, 5 minutes. Recovery run, 2 minutes. Repeat 3 times.
• Cool down, 5 to 10 minutes.
• Stretch.

── Day 3 
→ Rest
Rest day.

── Day 4 
→ W10D4-Easy Run
• Run, easy pace, 20 minutes.
• Cool down, 5 to 10 minutes.
• Stretch.

── Day 5 
→ Rest
Rest day.

── Day 6 
→ W10D6-Recovery Run
Optional: Swap Sunday's run to today.

── Day 7 
→ W10D7-5k Race
RACE DAY! GOOD LUCK!
```

Feel free to use this discussion thread if you have any question, suggestion, or if you just want to share your experience :) 

Enjoy ! 

Colin

**Comment by MaelaKloareg** [comment]
OK so all these weeks we were coached on pace (recovery, easy or steady), and the last instruction is just : "W10D7-5k Race, RACE DAY! GOOD LUCK!" :-)
What is your advice @ColinFay ? I would start on easy pace and push a bit after 20 min if if feel it's ok... but I have no idea if it's the best strategy.

**Reply by MaelaKloareg** [reply]
I've made it :-) ! I started faster than I expected but I was feeling well so I just trusted my feeling. I had to slow down a bit at the end, but I made it in 36:45 which whould have been absolutely incredible a few weeks ago (it still is actually ;-) ). I felt good, it was not painful. (NB : to make it easier I chose a perfectly flat trail, two 2.5km laps around a small lake)
Thank you so much for this training program @ColinFay , it was perfect for me (for my level and my motivation).
10k next :-) !

**Reply by ColinFay** [reply]
🎉 🎉 🎉 

so glad to hear you made it and that the program has motivated you!!

congrats !!

**Reply by MaelaKloareg** [reply]
Thanks !
